src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -994,11 +995,20 @@
                ExcludeEmptyDirectories="true"
                PackSymbolPackage="true"
                SymbolPackageOutputDirectory="$(SymbolPackageOutputPath)"
+               PackPrefixedSymbolPackage="$(PackPrefixedSymbolPackage)"
+               PrefixedSymbolPackageOutputDirectory="$(PrefixedSymbolPackageOutputPath)"
                AdditionalLibPackageExcludes="@(AdditionalLibPackageExcludes)"
                AdditionalSymbolPackageExcludes="@(AdditionalSymbolPackageExcludes)"
                Condition="'$(_SkipCreatePackage)' != 'true'"/>
-    <!-- Create a marker that records the path to the generated package -->
-    <WriteLinesToFile Lines="$(PackageOutputPath)$(Id).$(Version).nupkg;$(SymbolPackageOutputPath)$(Id).$(Version).symbols.nupkg"
+
+    <!-- Create markers that record the paths to the generated packages -->
+    <ItemGroup>
+      <OutputPackagePath Include="$(PackageOutputPath)$(Id).$(Version).nupkg" />
+      <OutputPackagePath Include="$(SymbolPackageOutputPath)$(Id).$(Version).symbols.nupkg" />
+      <OutputPackagePath Include="$(PrefixedSymbolPackageOutputPath)symbols.$(Id).$(Version).symbols.nupkg" />
```

Comment:
I agree, having an output would be better. The second `OutputPackagePath` is in the same boat, actually: sometimes symbol packages aren't created because there are no symbols to package. I didn't do it during the last PR because it doesn't hurt to list files here that don't exist, but getting rid of the duplication's a good point. I'll add this, thanks!